app/models/web_hook.rb

```diff
@@ -16,6 +18,7 @@ def self.specific
   end
 
   def fire(protocol, host_with_port, deploy_gem, version, delayed = true)
+    return if disabled?
```

Comment:
I think `fire` should not care about enabled/disabled option. Controller should do this.